src/praisonai-agents/praisonaiagents/mcp/mcp_server.py

```
@@ -244,14 +244,18 @@ def run_stdio(self) -> None:
     
     def run_sse(
         self,
-        host: str = "0.0.0.0",
+        host: "Optional[str]" = None,
         port: int = 8080,
         security: "Optional[SecurityConfig]" = None,
```

Ruff (0.15.21)

[error] Undefined name `SecurityConfig`

(F821)